packages/libvoikko/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE="http://voikko.sourceforge.net"
```

Comment:
Since this is an indirect dependency it would be good to mention what package requires it.
```suggestion
# Build time dependency for kf6-sonnet
TERMUX_PKG_HOMEPAGE="http://voikko.sourceforge.net"
```